test_go.py

```diff
@@ -19,5 +21,18 @@ def test_go_version(container):
 )
 def test_kured(container, container_git_clone):
     cmd = container.connection.run(container_git_clone.test_command)
-    print(cmd.stdout)
     assert cmd.rc == 0
+
+
+@pytest.mark.parametrize(
+    "host_git_clone",
+    [
+        GitRepositoryBuild(
+            repository_url="https://github.com/mvarlese/fleet",
+        ).to_pytest_param()
+    ],
+    indirect=["host_git_clone"],
+)
+def test_fleet(host, host_git_clone, dapper):
+    dest, _ = host_git_clone
+    host.run_expect([0], f"cd {os.path.join(dest, 'fleet')} && {dapper}")
```

Comment:
shouldn't this be dapper make ?

Reply:
In my local tests, I simply run "make" from the top-level folder in the fleet project indeed.

Reply:
It should be either `dapper build` or `dapper ci`.